xml/sle_update_upgrading.xml

```diff
@@ -369,7 +293,102 @@ tmpfs          506M     0  506M   0% /dev/shm
       <para>If your machine serves as a &vmhost; for &kvm; or &xen;,
         make sure to properly shut down all running &vmguest;s prior to
         the update. Otherwise you may not be able to access the guests
-        after the update. </para>
+        after the update.</para>
+    </sect2>
+  </sect1>
+
+  <sect1 xml:id="sec.update.disk">
+    <title>Partitioning and Disk Space</title>
+    <para>
+      Software tends to <quote>grow</quote> from version to
+      version. Therefore, take a look at the available partition space
+      before updating. If you suspect you are running short of disk space,
+      secure your data before increasing the available space
+      by resizing partitions, for example.
+
+      There is no general rule regarding how much space each partition
+      should have. Space requirements depend on your particular partitioning
+      profile and the software selected.
+    </para>
+
+    <note>
+      <title>Check for enough Space in &yast;</title>
```

Comment:
I know this one is not your fault... But you should use gerund for verbs in headlines [as long as it makes sense]: "Check*ing* for..."
There are more cases of this.